docs/index.md

```diff
@@ -1,3 +1,60 @@
-# GTR
+<div align="center">
+  <img src="assets/logo-trackers-violet.svg" alt="Trackers Logo" width="200" height="200">
+</div>
 
-A unified library for object tracking
+# Installation
+
+You can install `trackers` in a [**Python>=3.9**](https://www.python.org/) environment.
+
+!!! example "Basic Installation"
+
+    === "pip"
```

Comment:
@onuralpszr I created those commands based on `supervision` docs; will this work for `trackers` as well?

Reply:
Yes lgtm

Reply:
Also add deepsort as well